README.md

```diff
@@ -0,0 +1,77 @@
+# OTP Deploy
+
+This repo contains all the deploy scripts, CI/CD, and config files for the MBTA's OpenTripPlanner instance.
+
+## Setup
+You'll need to clone this repo run the following from the project root:
+1. `asdf install`
+1. `mbta/update_gtfs.sh` - fetches latest MBTA and Massport GTFS data
+1. `mbta/update_pbf.sh` - updates OpenStreetMap data
+1. `mbta/build.sh` - packages OTP into a jar, then runs the OTP build process
+
+If you want to test with your own local GTFS changes, put a copy of your GTFS file (if you've built
+one with [`gtfs_creator`](https://github.com/mbta/gtfs_creator), it will be written to
+`<gtfs_creator path>/output/google/MBTA_GTFS.zip`) in `var/MBTA_GTFS.gtfs.zip` and re-run the
+`build.sh` script.
+
+## Run Locally
+With all of that setup done, you should be able to run `mbta/server.sh`. This will start your local
+OTP instance and, when ready, print a message saying that the web server is ready and listening.
+
+Open a browser pointing to [`localhost:8080`](http://localhost:8080), and you'll have a bare-bones web interface to OTP where
+you can try out trip plans.
+
+## Updating OTP from upstream
+
+This repo uses git submodules to pin OTP to a specific commit and pull the code for use in the build process.
+To pull the latest OTP changes, do the following:
+
+1. `git submodule update --init --force --remote`
+1. Commit the updated submodule to a feature branch
+1. Push and test the update in dev
+1. Put up a PR and merge to master once reviewed
```

Comment:
__issue__: it seems like this is now out of date with the way that OTP is being fetched?